airbyte-integrations/connectors/destination-mongodb/src/main/java/io/airbyte/integrations/destination/mongodb/MongoDbDestinationUtils.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.mongodb;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.airbyte.db.jdbc.JdbcUtils;
+
+public class MongoDbDestinationUtils {
```

Comment:
Is there a possibility of combining this with MongoDbSourceUtils to prevent code duplication?

Reply:
Good point, initially I didn't do that simply because it was in different packages but looking at this right now I see there's a `MongoUtils.java` where these changes and your changes can live. I'll work on moving the changes over to that utility file